prometheus-postgres-exporter/templates/deployment.yaml

```diff
@@ -28,6 +28,8 @@ spec:
               value: {{ .Values.master | quote }}
             - name: DATA_SOURCE_USER
               value: {{ .Values.username | quote }}
+            - name: PG_EXPORTER_EXCLUDE_DATABASES
+              value: 'rdsadmin'
```

Comment:
rdsadmin should be a default values for the chart but we need to override it sometimes. So I think it's better to move it in `values.yaml`